stories/InterProUI/ProteinViewer.stories.tsx

```diff
@@ -3,11 +3,13 @@ import type { Meta, StoryObj } from '@storybook/react';
 
 import { ProteinViewer } from 'components/ProteinViewer';
 
-import Provider from './Provider';
-import configureStore from './configureStore';
+import Provider from '../Provider';
+import configureStore from '../configureStore';
 
 const store = configureStore();
 
+// TODO: There is a problem when loading this stories twice
```

Comment:
storybook breaks in the browser. And the console shows an error stack trace that starts with:
```
Uncaught TypeError: Converting circular structure to JSON
```
Something in the Protein Viewer is mutating the original data, probably the way we use d3. This is not an issue in the website because data always gets calculated from the API payload, so basically starts with new data every time it gets into a protein viewer. But in the story, it reuses the same variable, so the mutations persist.

Or at least that's my suspicion, I'm thinking we can use the same way we import the data for skylign, and that might solve the storybook issue. A better, but more time-consuming fix will be checking the components themselves to make their methods unmutable.

Reply:
Mhh, writing it in the old storybook format did the trick. I'm guessing that way it doesn't try to reuse the same variables.